blog/2015/02/weekly-update-2015-02-06.html

```diff
@@ -0,0 +1,68 @@
+  </tr>
+  <tr>
+    <td>Conversations:</td>
+    <td>"..what matters most in teaching is the act itself - the verb.." - <a href="{{page.root}}/blog/2015/02/nouns-and-verbs.html">Book recommendations to support great teaching </a></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/huang-scientific-coding-vs-software-engineering.html">Scientific coding and software engineering: what's the difference?</a> - Post by Daisie Huang - please add your comments</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/rewarding-software-sharing.html">Rewarding Software Sharing by Mapping Scientific Software</a> - Post by Chris Bogart - please share your ideas</td>
+  </tr>
+    <tr>
+    <td>Events:</td>
+    <td>SWC instructor training & Intro to Web Programming at <a href="{{page.root}}/blog/2015/02/workshops-in-march-at-lbl.html">Lawrence Berkeley Lab</a> - March 2015</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/ropensci-unhackathon.html">rOpenSci Unconference</a> - March 2015</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/ssi-collaborations-workshop.html">Software Sustainability Institute's Collaborations Workshop</a> - March 2015</td>
+  </tr>
+  <tr>
+    <td>Other News:</td>
+    <td><a href="{{page.root}}/blog/2015/02/welcome-our-newest-instructors.html">Eight new instructors joined SWC</a></td>
+  </tr>
+</table>
+<!-- end excerpt -->
+<p><strong>Details</strong></p>
+<ul>
+  <li>
+    This week the first meeting of the new SWF steering committee took place
+  </li>
+  <li>
+    Juan Nunez-Iglesias, Harriet Dashnow, and Stefan van der Walt are looking for code nominations to be used in their book called Elegant SciPy - <a href=http://ilovesymposia.com/2015/02/04/call-for-code-nominations-for-elegant-scipy/">please get in touch with them </a>
+  </li>
+ <li>
+    Sheldon McKay and Greg Wilson hosted a second post-workshop instructor debriefing attended by 10 instructors.    <td><a href="{{page.root}}/blog/2015/02/instructor-debriefing-2015-01-27.html">Read more about their lessons learnt</a>
+  </li>
+  <li>
+	Neil Davis summarised their experience and innovations introduced at the second SWC workshop held at <a href={{page.root}}/blog/2015/02/workshop-in-illinois.html">University of Illinois</a>. Their suggestions include: (i) get feedback at each break; (ii) introduce the idea that knowledge gained at SWC workshops can be applied tomorrow or on Monday as opposed to in future. He questioned whether SWC events advertisements are effective at creating a clear picture of what participants can expect.
```

Comment:
There is a missing quote before the link. Need to replace with

~~~
	Neil Davis summarised their experience and innovations introduced at the second SWC workshop held at <a href="{{page.root}}/blog/2015/02/workshop-in-illinois.html">University of Illinois</a>. Their suggestions include: (i) get feedback at each break; (ii) introduce the idea that knowledge gained at SWC workshops can be applied tomorrow or on Monday as opposed to in future. He questioned whether SWC events advertisements are effective at creating a clear picture of what participants can expect.
~~~